pkg/commands/process/settings/rules/ruby/lang/eval_using_user_input.yml

```diff
@@ -0,0 +1,57 @@
+patterns:
+  - pattern: |
+      RubyVM::InstructionSequence.compile($<USER_INPUT>$<...>)
+    filters:
+      - variable: USER_INPUT
+        detection: ruby_lang_eval_using_user_input_user_input
+  - pattern: |
+      $<_>.$<METHOD>($<USER_INPUT>$<...>)
+    filters:
+      - variable: METHOD
+        values:
+          - eval
+          - instance_eval
+          - class_eval
+          - module_eval
+      - variable: USER_INPUT
+        detection: ruby_lang_eval_using_user_input_user_input
+  - pattern: |
+      $<METHOD>($<USER_INPUT>$<...>)
+    filters:
+      - variable: METHOD
+        values:
+          - eval
+          - instance_eval
+          - class_eval
+          - module_eval
+      - variable: USER_INPUT
+        detection: ruby_lang_eval_using_user_input_user_input
+auxiliary:
+  - id: ruby_lang_eval_using_user_input_user_input
+    patterns:
+      - params
+      - request
+      - cookies
+      - | # AWS lambda
+        def $<_>($<!>event:, context:)
+        end
+languages:
+  - ruby
+trigger: presence
+severity:
+  default: high # FIXME
+metadata:
+  description: "Do not generate code using user input."
+  remediation_message: |
+    ## Description
+
+    TODO
+
+    ## Remediations
+    TODO
+    <!--
+    ## Resources
+    Coming soon.
+    -->
+  dsr_id: DSR-? # FIXME
```

Comment:
field is optional so dont specify if it dosen't have one

Reply:
are we dropping DSRs once we've added CWEs? I think it probably has an appropriate DSR but I just haven't added it yet.